public/locales/en/panel-gettingstartedtour.json

```diff
@@ -0,0 +1,200 @@
+{
+  "previous-button": "Previous",
+  "next-button": "Next",
+  "finish-button": "Finish",
+  "mouse-descriptions": {
+    "altitude": "The altitude is displayed at the top left corner of the screen. Right click and drag to go closer or further away.",
+    "navigation": "The latitude and longitude are displayed at the top left corner of the screen. Left click and drag to orbit."
+  },
+  "components": {
+    "clear-sky-button": {
+      "info": "Is it hard to find? Try clearing the sky",
```

Comment:
```suggestion
      "tooltip": "Is it hard to find? Try clearing the sky",
```
for consistency?

Reply:
This is not a tooltip, though. It's a text next to the button. 

This is a bit confusing, since the text is baked into the component named "ClearSkyButton" (i.e. the button does not only include the actual button

![image](https://github.com/user-attachments/assets/fb60f1bb-c0ab-4a18-97b7-4d4f96a65efc)

Reply:
Sounds good!